postgres/src/config.rs

```diff
@@ -90,6 +93,7 @@ use tokio_postgres::{Error, Socket};
 #[derive(Clone)]
 pub struct Config {
     config: tokio_postgres::Config,
+    notice_callback: Arc<dyn Fn(DbError) + Send>,
```

Comment:
This does mean that `Config` will no longer be `Sync` - do you think it'll be a problem to require the closure to be `Sync + Send` instead?

Reply:
Oh right, mpsc channels aren't sync :/ I guess we'll just do this and see if it breaks anyone then :(